BlockServer/runcontrol/runcontrol_manager.py

```diff
@@ -321,21 +320,21 @@ def wait_for_ioc_start(self, time_between_tries=2):
         while not started and loop_count < MAX_LOOPS_TO_WAIT_FOR_START:
```

Comment:
This whole syntax could be greatly improved by using a for-else loop:

```
for loop_count in range(MAX_LOOPS_TO_WAIT_FOR_START):
    if [ioc_restarting] or [still_waiting_for_ioc_to_start]:
        [sleep]
    else:
        self._rc_ioc_start_time = latest_ioc_start
        print_and_log("... Runcontrol IOC started")
        break
else:
    print_and_log("Runcontrol appears not to have started", "MAJOR")
...
```

Using a loop tracker like `started` is old hat and exactly what the `for-else` syntax is for.